### PR TITLE
Fix the example usage in docs/index.md

### PR DESCRIPTION
**Pull request by hrmsk66**

### CODE REVIEW

**Comment by smaeda-ks** [comment]
@hrmsk66 files under `/docs` are artifacts generated by the `make generate-docs` [command](https://github.com/fastly/terraform-provider-fastly#building-the-documentation), so the one we need to update is:
https://github.com/fastly/terraform-provider-fastly/blob/main/examples/index-fastly-provider.tf.tmpl

which is referenced by this template file:
https://github.com/fastly/terraform-provider-fastly/blob/main/templates/index.md.tmpl

Could you update that one and run `make generate-docs` then push another commit? Thanks for catching this!

**Comment by hrmsk66** [comment]
@smaeda-ks, ran the make command. Thanks for the comment.

File: examples/index-fastly-provider.tf.tmpl [hunk 1]
```diff
@@ -2,8 +2,9 @@
 terraform {
   required_providers {
     fastly = {
-      source = "fastly/fastly"
-      version >= "__VERSION__"
+      source  = "fastly/fastly"
+      version = ">= __VERSION__"
+
```
Comment (Integralist) [inline examples/index-fastly-provider.tf.tmpl]:
```suggestion
```

File: docs/index.md [hunk 2]
```diff
@@ -24,8 +24,9 @@ The Fastly provider prior to version v0.13.0 requires using
 terraform {
   required_providers {
     fastly = {
-      source = "fastly/fastly"
-      version >= "0.40.0"
+      source  = "fastly/fastly"
+      version = ">= 0.41.0"
+
```
Comment (Integralist) [inline docs/index.md]:
```suggestion
```

**Pull request merged**